bayesian_benchmarks/tasks/utils.py

```diff
@@ -0,0 +1,17 @@
+"""
+Helper functions...
+
+"""
+
+from typing import List
+
+import numpy as np
+from scipy.special import logsumexp
+
+def meansumexp(logps: List[np.ndarray]) -> np.ndarray:
```

Comment:
Given that this is a helper function I would make it more general by letting it accept an array (instead of a List of array's) and an axis on which to reduce. Namely,
`meansumexp(a: np.array, axis)`